services/order.py

```diff
@@ -0,0 +1,36 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from django.contrib.auth import get_user_model
+import datetime
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: dict,
+        username: str,
+        date: str = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
+
+    order = Order.objects.create(user=user)
+    if date:
+        _date = datetime.datetime.strptime(date, "%Y-%m-%d %H:%M")
```

Comment:
do you really need another one variable? You can overwrite your `date` as it's not used anymore.